orderer/consensus/smartbft/metrics.go

```diff
@@ -25,16 +25,26 @@ var (
 		LabelNames:   []string{"channel"},
 		StatsdFormat: "%{#fqname}.%{channel}",
 	}
+	isLeaderOpts = metrics.GaugeOpts{
```

Comment:
Can we also add the identifier of the leader, in another metric? e.g "the_leader".

Since this might be used by different organizations (in theory), the organization that is running a follower might wanna know who is the leader, but it cannot discern that from the IsLeader bit.